e2e-tests/spring-rest-opensearch/src/test/java/org/evomaster/e2etests/spring/rest/opensearch/age/OpenSearchAgeEMTest.java

```diff
@@ -0,0 +1,35 @@
+package org.evomaster.e2etests.spring.rest.opensearch.age;
+
+import com.foo.spring.rest.opensearch.age.OpenSearchAgeController;
+import org.evomaster.core.EMConfig;
+import org.evomaster.core.problem.rest.data.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.e2etests.utils.RestTestBase;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class OpenSearchAgeEMTest extends RestTestBase {
+
+    @BeforeAll
+    public static void initClass() throws Exception {
+        EMConfig config = new EMConfig();
+        config.setInstrumentMR_OPENSEARCH(true);
+        RestTestBase.initClass(new OpenSearchAgeController(), config);
+    }
+
+    @Test
+    public void testRunEM() throws Throwable {
+        runTestHandlingFlaky(
+            "OpenSearchAgeEM",
+            "org.foo.spring.rest.opensearch.OpenSearchAgeEM",
+            1000,
+            true,
+            (args) -> {
+                setOption(args, "instrumentMR_OPENSEARCH", "true");
+
+                Solution<RestIndividual> solution = initAndRun(args);
+
+//                assertHasAtLeastOne(solution, HttpVerb.GET, 404, "/age/{q}", null);
```

Comment:
why no assertions? if the techniques work, when you should be able to cover the different endpoints you have designed. if not yet, and tehcniques needs more fixing, it is ok to have a PR now, but: (1) add the needed assertions on the different endpoints; (2) mark the test with `@Disabled`; (3) leave a TODO comment stating this is failing and need more work